Makefile

```diff
@@ -775,19 +729,14 @@ check_links:
 	@#  --ignore-url=https://github.com/zama-ai/concrete-ml-internal/issues: because issues are
 	@#		private
 	@#	--ignore-url=.gitbook/assets : some gitbook functionalities use links to images to include
```

Comment:
you removed `--ignore-url=.gitbook/assets` but you kept a comment about it  

same for `--ignore-url=_static/webpack-macros.html` above